### PR TITLE
Made requested change by GKE to meet expectation

### PR DESCRIPTION
**Pull request by mengdong**
1: remove use of istio and use simple ingress solution
2, update hpa version
3, usage of service account.

### CODE REVIEW

File: deploy/gke-marketplace-app/server-deployer/schema.yaml [hunk 1]
```diff
@@ -89,7 +89,7 @@ properties:
   modelRepositoryPath:
     type: string
     title: Bucket where models are stored. Please make sure the user/service account to create the GKE app has permission to this GCS bucket. Read Triton documentation on configs and formatting details, supporting TensorRT, TensorFlow, Pytorch, Onnx ... etc.
-    default: gs://triton_sample_models/models
+    default: gs://triton_sample_models/23_02
```
Comment (Tabrizian) [inline deploy/gke-marketplace-app/server-deployer/schema.yaml:92]: Does the models change with every release? Also, is the GCS bucket going to be created for every release before we publish on NGC?

CC @mc-nv

**Comment by mengdong** [comment]
rebased

**Comment by mengdong** [comment]
@nv-kmcgill53 @rmccorm4 any updates? Thanks!

File: deploy/gke-marketplace-app/README.md [hunk 2]
```diff
@@ -119,33 +126,31 @@ gcloud beta container node-pools create accel \
...
 
 We want to discuss HPA autoscaling metrics users can leverage. GPU Power(Percentage of Power) tends to be a reliable metric, especially for larger GPU like V100 and A100. GKE currently natively support GPU duty cycle which is GPU utilization in `nvidia-smi`. We ask users always profile their model to determine the autoscaling target and metrics. When attempting to select the right metrics for autoscaling, the goal should be to pick metrics based on the following: 1, meet SLA rrequirement. 2, give consideration to transient request load, 3, keep GPU as fully utilized as possible. Profiling comes in 2 aspects: If user decided to use Duty Cycle or other GPU metric, it is recommend establish baseline to link SLA requirement such as latency with GPU metrics, for example, for model A, latency will be below 10ms 99% of time when Duty Cycle is below 80% utilized. Additionally, profiling also provide insight to model optimization for inference, with tools like [Nsight](https://developer.nvidia.com/nsight-systems).
 
-As the application deployed successfully, get Istio Ingress host and port
+As the application deployed successfully, get the public ip from ingress
```
Comment (rmccorm4) [inline deploy/gke-marketplace-app/README.md:149]:
```suggestion
Once the application is deployed successfully, get the public ip from ingress:
```

File: deploy/gke-marketplace-app/trt-engine/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+## Description
+
+```
+docker run --gpus all -it --network host \
+    --shm-size=1g --ulimit memlock=-1 --ulimit stack=67108864 \
+    -v ~:/scripts nvcr.io/nvidia/tensorrt:23.02-py3 
+
+pip install onnx six torch tf2onnx tensorflow 
+# pip uninstall protobuf 
+# pip install protobuf==3.20.2
```
Comment (rmccorm4) [inline deploy/gke-marketplace-app/trt-engine/README.md:40]: Are these comments needed?

**Review by rmccorm4 — COMMENTED** [review]
Nice changes, mostly just typos/formatting suggestions

**Comment by mengdong** [comment]
thanks for the changes @rmccorm4

**Comment by rmccorm4** [comment]
@mengdong LGTM, just need to fix the merge conflicts

**Comment by mengdong** [comment]
@rmccorm4 just resolved the merge conflict, thanks!

**Comment by rmccorm4** [comment]
Lastly @mengdong, is there any reason why we use a TensorRT model for this example? I feel like it adds unnecessary complexity for the user. i.e. if we used an ONNX model or something more portable, the public demo bucket wouldn't have to be updated every release. This would also simplify our own upkeep and making sure the example works more broadly. What do you think?

And if TRT is an important part to demo, we could just keep the documentation on how users would create their own TRT model from the ONNX model for reference.

**Comment by mengdong** [comment]
Thanks @rmccorm4, I chose TRT mostly out of performance consideration and TRT provide best performance with Triton.

We do provide a section (README) on how to convert the onnx model to TRT. 

For our upkeep, it should very easy process (could be automated) following the README to update the TRT model in the bucket.

**Comment by rmccorm4** [comment]
> For our upkeep, it should very easy process (could be automated) following the README to update the TRT model in the bucket.

The extra restriction for only working on T4 machines can be cumbersome as well, but if no one's complained so far then I guess there's no reason to change.

**Review by rmccorm4 — APPROVED** [review]

**Comment by mengdong** [comment]
any updates... ?

File: deploy/gke-marketplace-app/client-sample/locustfile_bert_large.py [hunk 4]
```diff
@@ -67,4 +67,4 @@ def on_start(self):
             self.data = json.load(f)
 
         self.url1 = '{}/v2/models/{}/infer'.format(self.environment.host,
-                                                   'bert_large')
+                                                   'bert')
```
Comment (nv-kmcgill53) [inline deploy/gke-marketplace-app/client-sample/locustfile_bert_large.py:70]: nit: Should the file name change from `locustfile_bert_large.py` to `locustfile_bert.py`?

**Review by nv-kmcgill53 — APPROVED** [review]

File: deploy/gke-marketplace-app/client-sample/locustfile_bert_large.py (hunk 4 above)
Comment (mengdong) [reply deploy/gke-marketplace-app/client-sample/locustfile_bert_large.py:70]: fixed

**Review by rmccorm4 — APPROVED** [review]

**Comment by mengdong** [comment]
any more updates... ?

**Review by nv-kmcgill53 — APPROVED** [review]

**Pull request merged**